tests/Integration/FallbackLocaleTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * @copyright 2015-2016 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\Tests\Integration;
+
+use Contentful\Delivery\Client;
+
+/**
+ * The fallback chain set up is the following
+ * af -> zu-ZA -> en-US -> NULL
+ * ne-NP -> NULL
+ * bs_BA -> en-US -> NULL
+ */
+class FallbackLocaleTest extends \PHPUnit_Framework_TestCase
+{
+    private function createClient()
+    {
+        $client = new Client('irrelevant', '7dh3w86is8ls');
+
+        // First we load the Space
+        $client->reviveJson('{"sys": {"type": "Space","id": "7dh3w86is8ls"},"name": "Fallback Locales","locales": [{"code": "en-US","default": true,"name": "U.S. English","fallbackCode": null},{"code": "af","default": false,"name": "Afrikaans","fallbackCode": "zu-ZA"},{"code": "zu-ZA","default": false,"name": "Zulu (South Africa)","fallbackCode": "en-US"},{"code": "bs-BA","default": false,"name": "Bosnian (Bosnia and Herzegovina)","fallbackCode": "en-US"},{"code": "ne-NP","default": false,"name": "Nepali (Nepal)","fallbackCode": null}]}');
+
+        // then the Content Type
+        $client->reviveJson('{"sys": {"space": {"sys": {"type": "Link","linkType": "Space","id": "7dh3w86is8ls"}},"id": "sampleType","type": "ContentType","createdAt": "2016-06-21T08:53:55.767Z","updatedAt": "2016-06-21T13:16:34.507Z","revision": 5},"displayField": "title","name": "Sample Type","description": "","fields": [{"id": "title","name": "title","type": "Symbol","localized": true,"required": true,"disabled": false,"omitted": false}]}');
+
+        return $client;
+    }
+
+    /**
+     * No ne-NP
```

Comment:
I'm now not anymore sure, if I like the naming of the locales here. Do you mind writing what no ne-NP means?